### PR TITLE
added some bootstrap for the camperCell

### PR DESCRIPTION
**Pull request by benjithorpe**
<h1>Camper Cells</h1>

<p>They were all in a column for medium screen sizes</p>

Screenshot (4)

<p>They are all in a 2 column format for medium screen</p>

Screenshot (5)

### CODE REVIEW

File: puzzles/puzzle1.html [hunk 1]
```diff
@@ -130,79 +138,83 @@ <h1>Puzzle #1</h1>
...
         <p>
           "Many of life's failures are people who did not realize how close they were to success when they gave up." -Thomas A. Edison
         </p>
       </div>
     
+      <div class="camperCell col-md-5 col-lg-2" id="sparrow">
+        <h1>Benjamin Thorpe</h1>
+        <img src="https://avatars0.githubusercontent.com/u/61359802?v=4" alt="benjamin">
+
+        <p><q>Stay Hungry, Stay Foolish</q> - Steve Jobs</p>
```
Comment (DavidMatthewFraser) [inline puzzles/puzzle1.html:215]: Good one 😄

File: puzzles/puzzle1.html [hunk 2]
```diff
@@ -130,79 +138,83 @@ <h1>Puzzle #1</h1>
...
+      <li>Inside your div, add a p element that contains your favorite quote</li>
+      <li>Add any custom styling by targeting your custom id in the provided style tag</li>
     </ol>
   </div>
 
   <div id="yourWork">
     <h1>Camper Cells</h1>
 
-    <div id="cellPanel">
+    <div id="cellPanel" class="row p-4">
```
Comment (DavidMatthewFraser) [inline puzzles/puzzle1.html:159]: Great use of the bootstrap grid! This needed to be addressed and I'm glad you did it so cleanly!

File: index.html [hunk 3]
```diff
@@ -19,6 +19,7 @@
           <img id="logo" src="assets/img/e2g.png" />
           <h3>Easy to HTML</h3>
         </div>
+
```
Comment (DavidMatthewFraser) [inline index.html:22]: I agree with the spacing

File: index.html [hunk 4]
```diff
@@ -45,17 +47,19 @@ <h3>Easy to HTML</h3>
             </form>
           </div>
           
-        <a href='puzzles/puzzle1.html'><button class='btn btn-deault'>Puzzle One Prototype</button></a>
-          
+        <a href='puzzles/puzzle1.html'><button class='btn btn-default'>Puzzle One Prototype</button></a>
```
Comment (DavidMatthewFraser) [inline index.html:50]: Good Catch 👍

File: puzzles/puzzle1.html (hunk 1 above)
Comment (benjithorpe) [reply puzzles/puzzle1.html:215]: just trying my best, while am still learning some tough bootstrap concepts

**Pull request merged**